sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -689,6 +689,7 @@ object functions extends LegacyFunctions {
    * @group normal_funcs
    * @since 1.4.0
    */
+  @scala.annotation.varargs
   def array(colName: String, colNames: String*): Column = {
```

Comment:
when we have this annotation, can we just use one parameter `colNames: String*`?